src/widgets/programs/programs-form.component.tsx

```diff
@@ -229,7 +229,7 @@ export default function ProgramsForm(props: ProgramsFormProps) {
                   onChange={evt => setProgram(evt.target.value)}
                   required
                 >
-                  <option>{t("Choose a program")}:</option>
+                  <option>{t("chooseAProgram", "Choose a program")}:</option>
```

Comment:
```suggestion
                  <option>{t("chooseProgram", "Choose a program")}:</option>
```

These are like variable names, so you can drop particles